Multisig/UI/Settings/Passcode/SecuritySettingsViewController.swift

```diff
@@ -225,6 +210,33 @@ class PasscodeSettingsViewController: UITableViewController {
         }
     }
 
+    // Handle new secirtiy method
+    // Change lock method
+    // Update security center storing protection
+    // activate biometrics with new lock method
+
+    private func toggleBiometricsSecurityLock(newLockMethod: LockMethod) {
+        guard newLockMethod != App.shared.securityCenter.lockMethod else { return }
+        withPasscodeAuthentication(for: "Login with biometrics") { [unowned self] success, nav, finish in
+            let completion = { [unowned self] in
+                finish()
+                reloadData()
+            }
+
+            if success {
```

Comment:
I thought that this call is not needed for the new system, because the securityCenter will request passcode by itself, so I think this can be deleted.